editor/state.js

```diff
@@ -148,6 +148,15 @@ export function mode( state = 'visual', action ) {
 	return state;
 }
 
+export function sidebar( state = { opened: false }, action ) {
```

Comment:
What other properties do you expect this to hold? Wondering if a simpler `isSidebarOpened` reducer with a boolean value might be more appropriate for now.